### PR TITLE
Helm collector type

### PR DESCRIPTION
**Pull request by prash897**
Helm- Change Release Details
Is this a Change or Addition: An Addition
What has been added: An enum "Helm" is added to CollectorType

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
     <groupId>com.capitalone.dashboard</groupId>
     <artifactId>core</artifactId>
     <packaging>jar</packaging>
-    <version>3.7.11</version>
+    <version>3.7.12</version>
```
Comment (rvema) [inline pom.xml:22]: please change to 3.7.13

**Comment by rvema** [comment]
Also please update the branch

**Comment by prash897** [comment]
Changes Addressed

**Comment by rvema** [comment]
@prash897 please resolve the merge conflicts

**Comment by prash897** [comment]
Changes Addresses

**Comment by prash897** [comment]
Removed

**Comment by rvema** [comment]
@prash897 please use hygieia-core version 3.7.14 for your helm collector

**Pull request merged**